feat(mouse): Add mouse move and scroll support (#2477)
zmkfirmware/zmk: docs/docs/config/pointing.md

| Property           | Type          | Description                                                         |
| ------------------ | ------------- | ------------------------------------------------------------------- |
| `device`           | phandle       | Input device handle                                                 |
| `input-processors` | phandle-array | List of input processors (with parameters) to apply to input events |

#### Child Properties

Additional properties can be set on child nodes, which allows changing the settings when certain layers are enabled:

| Property           | Type          | Description                                                                                      |
| ------------------ | ------------- | ------------------------------------------------------------------------------------------------ |
| `layers`           | array         | List of layer indexes. This config will apply if any layer in the list is active.                |
| `input-processors` | phandle-array | List of input processors (with parameters) to apply to input events                              |

> | `process-next`     | bool          | Whether to first apply the base input processors before the processors specific to this override |

Reviewer: ```suggestion
| `process-next`     | bool          | Whether to continue applying other input processors after this override if it takes effect |
```
(will need a prettier pass)
